assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.test.jsx

```diff
@@ -53,11 +55,19 @@ describe('SapSystemsOverviews component', () => {
         database_sid: attachedRdbms,
       } = sapSystem;
 
+      const sapSystemType = 'ABAP';
+      const modifiedApplicationInstances = applicationInstances.map(
```

Comment:
Do we need to modify the variable after creating?
Did you try something like this above:
```
      const sapSystem = sapSystemFactory.build({
        ensa_version: 'ensa1',
        application_instances: applicationInstanceFactory.buildList(2, { features: 'ABAP' }),
      });
```

Reply:
No not really, overwriting makes more sense :)